### PR TITLE
[web][camera] Fix missing function "getCapabilities" in Firefox

### PR DESCRIPTION
**Pull request by miso-belica**
# Why
As explained in https://github.com/expo/expo/issues/13182 Firefox freezes when this happens which is quite annoying.

# How
I skip calling the affected function entirely. It would result having no effect anyway of I pretend to have no capabilities.

# Test Plan
All details were already provided in https://github.com/expo/expo/issues/13182

# Checklist
- [x] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [x] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [x] This diff will work correctly for `npx expo prebuild` & EAS Build (eg: updated a module plugin).

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206749693628155

### CODE REVIEW

**Review by lukmccall — COMMENTED** [review]
Looks good! Thanks for your contribution 🎊 
Can you add the changelog entry for your change? The changelog file is inside the `expo-camera` package.

**Comment by miso-belica** [comment]
Yes, sure. Sorry it took a while. I hope it OK now.

File: packages/expo-camera/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@
 ### 🎉 New features
 
 ### 🐛 Bug fixes
+- On `web`, fix missing function "getCapabilities" in Firefox. ([#28947](https://github.com/expo/expo/pull/28947) by [@miso-belica](https://github.com/miso-belica))
```
Comment (lukmccall) [inline packages/expo-camera/CHANGELOG.md:10]:
```suggestion

- On `web`, fix missing function "getCapabilities" in Firefox. ([#28947](https://github.com/expo/expo/pull/28947) by [@miso-belica](https://github.com/miso-belica))
```

**Review by lukmccall — APPROVED** [review]
Looking good! I'll merge it as soon as the CI passes

**Pull request merged**